analysis/utility.py

```diff
@@ -0,0 +1,53 @@
+#/usr/bin/python 
+# npagane | python utilities file
+
+import pathlib
+import numpy as np
+
+# set parameters
+lengthPerBP = 0.332
+defaultOmega = 34.3*np.pi/180
+defaultDirectory = str(pathlib.Path(__file__).parent.absolute()).split(__file__)[0]+'/../'
+# read in translation and rotation files
+nucleosomeTran = np.loadtxt(defaultDirectory+'input/nucleosomeT')
+nucleosomeRot = np.loadtxt(defaultDirectory+'input/nucleosomeR')
+
+# define bead translation + rotation
+def rotateBead(Uin,Vin,Rin,linkBP,wrapBP):
```

Comment:
Also, what do each of these parameters do? what is their type? should be in docstring.